versions.tf

```diff
@@ -1,4 +0,0 @@
-
-terraform {
```

Comment:
I think we want to keep this around.

Reply:
I thought we were walking away from this. But I will add back.